### PR TITLE
New GUI Test project

### PR DESCRIPTION
**Pull request by KostinP**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/326

### CODE REVIEW

File: GUITestProject/Program.cs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            {
+                process.Start();
+
+                _logger.LogInformation($"Command '{command}' successfully invoked for proccess named {process.ProcessName}");
+            }
+            catch (Exception e)
+            {
+                _logger.LogWarning($"Command '{command}' failed invoked for proccess named {process.ProcessName}");
+
+                _logger.LogWarning(e, e.Message);
```
Comment (borovHokage) [inline GUITestProject/Program.cs:148]: is it necessary to use two logger field?

File: GUITestProject/Program.cs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+namespace GUITestProject
+{
+    public class Program
+    {
+        private static ILogger<Program> _logger;
+        private static List<(string ServiceName, string Url)> _servicesInfo;
+        private static List<Process> _runningServices = new();
+
+        private static string ServicesPath => Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..\\..\\..\\..\\"));
```
Comment (BelehovEgor) [inline GUITestProject/Program.cs:17]: AppDomain.CurrentDomain.BaseDirectory, "..\\..\\..\\..\\") - what does it mean?

**Review by djino97 — APPROVED** [review]

File: GUITestProject/Program.cs (hunk 2 above)
Comment (KostinP) [reply GUITestProject/Program.cs:17]: Climb 4 directories up, start in the same place where exe file

**Review by TIove — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

File: LT.DigitalOffice.Tests.sln [hunk 3]
```diff
@@ -14,10 +14,25 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
...
 Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Release|Any CPU.Build.0 = Release|Any CPU
```
Comment (MariyaSin) [inline LT.DigitalOffice.Tests.sln:28]: where do you get this Guid and what's going on here?

**Review by MariyaSin — APPROVED** [review]

File: LT.DigitalOffice.Tests.sln (hunk 3 above)
Comment (KostinP) [reply LT.DigitalOffice.Tests.sln:28]: It was added automatically. It's just adding a new project to the solution.

If you want to understand in more detail how the sln works, you can read this 
https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019
http://raph.es/blog/2011/04/targeting-platforms-in-visual-studio/

**Pull request merged**